jcsv.go

```diff
@@ -1,11 +1,84 @@
 package jcsv
 
+import (
+	"encoding/json"
+	"strconv"
+	"strings"
+)
+
 func JsonToCsv(j []byte, addHeaders bool) ([]byte, error) {
-	// TODO: convert JSON data in `j` into CSV format and return
-	return nil, nil
+	//create file object
+	var JSON file
+
+	//convert data in j to Data
+	json.Unmarshal(j, &JSON.Data)
+
+	//return converted CSV in []byte form
+	return JSON.Csv(addHeaders), nil
 }
 
 func CsvToJson(c []byte, hasHeaders bool) ([]byte, error) {
-	// TODO: convert CSV data in `c` into JSON format and return
-	return nil, nil
+
+	//convert []byte to string
+	CSVStr := string(c)
+
+	//split the string on the basis of '\n'
+	splittedStr := strings.Split(CSVStr, "\n")
```

Comment:
You can also do it like this:
```suggestion
	//split the string on the basis of '\n'
	splittedStr := strings.Split(string(c), "\n")
```

or `bytes.Split()` is also available.

Reply:
all right, that looks more cleaner too.